lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosResult.cxx

```diff
@@ -104,11 +104,145 @@ String FunctionalChaosResult::__repr__() const
+{
+  const String intermediateString = OSS() << string;
+  const UnsignedInteger columnWidth = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintColumnWidth");
+  const UnsignedInteger rightPadding = intermediateString.size() < columnWidth ? columnWidth - intermediateString.size() : 0;
+  OSS oss;
+  oss << intermediateString << String(rightPadding, ' ');
+  return oss;
+}
+
+String FunctionalChaosResult::__str__(const String & /*offset*/) const
 {
   OSS oss(false);
-  oss << "meta model=" << metaModel_ << Os::GetEndOfLine() << offset;
-  oss << "orthogonal basis=" << orthogonalBasis_;
+  const UnsignedInteger indicesSize = I_.getSize();
+  const EnumerateFunction enumerateFunction(orthogonalBasis_.getEnumerateFunction());
+  const UnsignedInteger outputDimension = metaModel_.getOutputDimension();
+  const UnsignedInteger inputDimension = distribution_.getDimension();
+  oss << getClassName() << "\n"
+      << "- input dimension=" << inputDimension << "\n"
+      << "- output dimension=" << outputDimension << "\n"
+      << "- distribution dimension=" << distribution_.getDimension() << "\n"  
+      << "- transformation=" << transformation_.getInputDimension() << " -> " << transformation_.getOutputDimension() << "\n"
+      << "- inverse transformation=" << inverseTransformation_.getInputDimension() << " -> " << inverseTransformation_.getOutputDimension() << "\n"
+      << "- orthogonal basis dimension=" << orthogonalBasis_.getMeasure().getDimension() << "\n"
+      << "- indices size=" << indicesSize << "\n"
+      << "- relative errors=" << relativeErrors_ << "\n"
+      << "- residuals=" << residuals_ << "\n";
+  oss << "\n";
+
+  const Scalar ell_threshold = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisThreshold");
+  const UnsignedInteger ell_size = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintEllipsisSize");
+  const UnsignedInteger columnWidth = ResourceMap::GetAsUnsignedInteger("FunctionalChaosResult-PrintColumnWidth");
+  const UnsignedInteger ellipsis = (indicesSize * outputDimension > ell_threshold);
+  const bool isEllipsisEnabled = (ellipsis and outputDimension > 2 * ell_size);
+  // Print table header
+  oss << "| Index |" << padString(" Multi-index ") << "|";
+  String intermediateString;
+  if (isEllipsisEnabled)
+  {
+    for (UnsignedInteger j = 0 ; j < ell_size; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    intermediateString = OSS() << " ... ";
+    oss << padString(intermediateString) << "|";
+    for (UnsignedInteger j = outputDimension - ell_size ; j < outputDimension; ++ j)
+    {
+      intermediateString = OSS() << " Coeff.#" << j;
+      oss << padString(intermediateString) << "|";
+    }
+    oss << "\n";
+  }
+  else
+  {
+      if (outputDimension == 1) 
+      {
+        intermediateString = OSS() << " Coefficient ";
```

Comment:
There is no need for an intermediate string here. Just use the string " Coefficient ".

Reply:
Fixed at a3b84d52d0a1a8f8efd09354300f7de1af3f1a5e